src/faldbt/lib.py

```diff
@@ -164,6 +164,12 @@ def _build_table_from_parts(
 ):
     from dbt.contracts.relation import Path, RelationType
 
+    # Avoids automatic quoting by Snowflake
+    if adapter.type() == "snowflake":
+        database = database.lower()
+        schema = schema.lower()
+        identifier = identifier.lower()
+
```

Comment:
Can we do this in the `_write_relation` function instead? I think this applies to any table to create with sqlalchemy, not the temporal one only

Reply:
I think we should also change the temporal table to have the `__f__` in lowercase. To remove the "hack" we had introduced (it does not really make a difference, but at least tests it for snowflake, right?)